README.md

```
@@ -0,0 +1,48 @@
+
+The user is responsible for consuming the `datalint` library to
+
+- specify how to parse the input file
+- specify how to extract the application name and version from the input file
+- create a layout file (patch) that defines their expected layout
+- create a rules file (patch) that defines their expected rules
+
+## Developer Get Started
+1. Clone the repo
+1. Install the requirements for the python virtual environment
+   ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)